shared/commonprops/commonProps.ts

```diff
@@ -21,11 +29,29 @@ export async function downloadModel() {
         var result = await init(resp + '/model.tflite', false);
```

Comment:
This is inside a for loop which means we will be logging telemetry events multiple times for the same thing.

Reply:
We should log once after retries count is exhausted